packages/vaadin-grid/src/vaadin-grid-column.js

```diff
@@ -147,13 +171,12 @@ export const ColumnBaseMixin = (superClass) =>
         '_widthChanged(width, _headerCell, _footerCell, _cells.*)',
         '_frozenChanged(frozen, _headerCell, _footerCell, _cells.*)',
         '_flexGrowChanged(flexGrow, _headerCell, _footerCell, _cells.*)',
-        '_pathOrHeaderChanged(path, header, _headerCell, _footerCell, _cells.*, renderer, headerRenderer, _bodyTemplate, _headerTemplate)',
         '_textAlignChanged(textAlign, _cells.*, _headerCell, _footerCell)',
         '_orderChanged(_order, _headerCell, _footerCell, _cells.*)',
         '_lastFrozenChanged(_lastFrozen)',
-        '_setBodyTemplateOrRenderer(_bodyTemplate, renderer, _cells, _cells.*)',
```

Comment:
I considered it would be better to rename these methods. The main reason is that they all render renderers/templates for a set of cells. Assigning a renderer or a template to a cell is only a detail of the implementation.

So now it looks more natural when using them in the following case:

https://github.com/vaadin/web-components/pull/1967/files#diff-72e5ef8375a545ca777f44fe856611e1cdd4d5b0e3f4e40b9b34783f73101896R182